src/js/controller/controller.js

```diff
@@ -606,14 +604,18 @@ Object.assign(Controller.prototype, {
         }
 
         function _seek(pos, meta) {
-            if (_model.get('state') === STATE_ERROR) {
+            const state = _model.get('state');
+            if (state === STATE_ERROR) {
                 return;
             }
             _programController.position = pos;
-            if (!_model.get('scrubbing') && _model.get('state') !== STATE_PLAYING) {
+            if (!_model.get('scrubbing') && state !== STATE_PLAYING) {
                 meta = meta || {};
                 meta.startTime = pos;
-                this.play(meta);
+
+                if (_model.get('dvrLive')) {
+                    this.play(meta);
```

Comment:
why do we play on dvr ?

Reply:
@karimMourra ```play``` is called when ```dvrLive``` is true for cases when we press the live button to get the live edge. The stream should be seeked to the live edge and then resume playback